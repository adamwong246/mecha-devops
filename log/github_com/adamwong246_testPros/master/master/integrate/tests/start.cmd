run,-d=true,github_com-adamwong246_testpros/master:master,npm,run,start