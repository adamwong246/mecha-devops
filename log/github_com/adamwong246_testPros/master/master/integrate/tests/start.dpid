cf7d349dc42a570fa8a5ace4d6493e0487deffb5a16ef805c186e9a9ca0cfd0e