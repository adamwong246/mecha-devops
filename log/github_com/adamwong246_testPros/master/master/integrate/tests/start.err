npm info it worked if it ends with ok
npm info using npm@3.10.10
npm info using node@v6.10.0
npm info lifecycle test-proj@1.0.0~prestart: test-proj@1.0.0
npm info lifecycle test-proj@1.0.0~start: test-proj@1.0.0
