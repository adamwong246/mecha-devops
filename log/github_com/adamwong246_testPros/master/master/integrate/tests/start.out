
> test-proj@1.0.0 start /usr/src/app
> node server.js

Running on http://localhost:8080
